opencti-platform/opencti-graphql/config/default.json

```diff
@@ -11,7 +11,8 @@
       "NEW_IMPORT_SCREENS",
       "FILIGRAN_LOADER",
       "CONTAINERS_AUTHORIZED_MEMBERS",
-      "TELEMETRY_COUNT_ACTIVE_USERS"
+      "TELEMETRY_COUNT_ACTIVE_USERS",
+      "DRAFT_WORKSPACE"
```

Comment:
small suggestion to move "DRAFT_WORKSPACE" a bit upper in the list (to avoid conflits with master)